### ISSUE TITLE
Perhaps add a log message about the chown command at startup

### ISSUE THREAD

**Issue by 9oOzv** [Wonderfall/docker-rtorrent-flood#20]
The chown command at startup seems to take a considerable amount of time (20-30 seconds for me). I think it would be helpful if there was a log message indicating that the container is indeed doing something and that you just need to wait.

**Comment by benjaminclot** [comment]
Or make the chown an option? Because when you have lots of files it takes forever and is not necessarily required.

**Comment by Wonderfall** [comment]
Yes, I'm gonna refactor the image soon so it won't be necessary.
Chown is fast on everything except `overlay2fs` unfortunately.
